templates/tailwind.config.js

```diff
@@ -1,12 +1,27 @@
 module.exports = {
   future: {
+    defaultLineHeights: true,
+    purgeLayersByDefault: true,
     removeDeprecatedGapUtilities: true,
-    purgeLayersByDefault: false
+    standardFontWeights: true
+  },
+  purge: {
```

Comment:
I remember there being an issue with Tailwind not purging with the built in purge which is why I moved it to the postcss config. Perhaps @andrewmcodes could shed some light.

Reply:
I can confirm on netlify it works with NODE_ENV=production

Reply:
![image](https://user-images.githubusercontent.com/4352208/101876341-8c493400-3b8c-11eb-8dfb-f22e7da0a7c6.png)

yep, definitely purges 😅